### PR TITLE
Add events to Stage

### PR DESCRIPTION
**Pull request by liajoy**
#85

### CODE REVIEW

**Comment by michalochman** [comment]
@liajoy can you add unit tests for that in `Stage.test.js` and `props.test.js`?

**Comment by liajoy** [comment]
@michalochman Test and code pretty have been done. Please take a look.

File: test/props.test.js [hunk 1]
```diff
@@ -36,4 +36,38 @@ describe("props", () => {
       });
     });
   });
+
+  describe("EVENT_PROPS", () => {
+    it("contain all DisplayObject events", () => {
```
Comment (michalochman) [inline test/props.test.js]: `it("should be a list containing all DisplayObject events", () => {`

**Comment by michalochman** [comment]
Looks good @liajoy! Please just change one test name and I'll merge&release it!

File: test/props.test.js (hunk 1 above)
Comment (liajoy) [reply test/props.test.js]: got it

**Pull request merged**